dashboards-observability/server/adaptors/custom_panels/custom_panel_adaptor.ts

```diff
@@ -71,12 +71,20 @@ export class CustomPanelsAdaptor {
         objectType: 'operationalPanel',
         maxItems: 10000,
       });
-      return response.observabilityObjectList.map((panel: any) => ({
-        name: panel.operationalPanel.name,
-        id: panel.objectId,
-        dateCreated: panel.createdTimeMs,
-        dateModified: panel.lastUpdatedTimeMs,
-      }));
+      var panelList: any[] = [];
+      for (const panel of response.observabilityObjectList) {
+        var object: CustomPanelListType = {
```

Comment:
use `const`? also might be better to avoid `object` since `Object` is a builtin word in js